modules/ocf/manifests/packages/shell.pp

```diff
@@ -0,0 +1,16 @@
+class ocf::packages::shell {
+  package { ['bash', 'tcsh', 'zsh']: }
+
+  file {
+    '/etc/bash.bashrc':
+      source => 'puppet:///modules/ocf/shell/bash.bashrc';
```

Comment:
during initial install, your file might be put in place before bash installs, then bash will install and want to replace that file (i think it probably won't for noninteractive installs though, so it's probably nbd)